common/lib/xmodule/xmodule/video_module/video_xfields.py

```diff
@@ -161,7 +161,7 @@ class VideoFields(object):
     )
     edx_video_id = String(
         help=_("If you were assigned a Video ID by edX for the video to play in this component, enter the ID here. In this case, do not enter values in the Default Video URL, the Video File URLs, and the YouTube ID fields. If you were not assigned an edX Video ID, enter values in those other fields and ignore this field."),
```

Comment:
We should discuss with the doc team whether we can get rid of the edX references in this help string. @lamagnifica ?